interp/constrexpr_ops.ml

```diff
@@ -616,8 +616,9 @@ let interp_univ_constraints env evd cstrs =
     let cstrs' = Univ.Constraint.add cstr cstrs in
     try let evd = Evd.add_constraints evd (Univ.Constraint.singleton cstr) in
         evd, cstrs'
-    with Univ.UniverseInconsistency e ->
-      CErrors.user_err ~hdr:"interp_constraint"
+    with Univ.UniverseInconsistency e as exn ->
+      let _, info = Exninfo.capture exn in
+      CErrors.user_err ~hdr:"interp_constraint" ~info
```

Comment:
@SkySkimmer could we replace this by the printer in `Himsg` ?

Reply:
Don't know tbh
To try just do eg `Definition foo@{|Set < Set} := 0.`

Reply:
Outputs seem identical modulo the `Set Printing Universes` option which has to be enabled. I wonder tho what the difference between:
```
        Univ.explain_universe_inconsistency UnivNames.pr_with_global_universes i
```
and
```
        Univ.explain_universe_inconsistency UnivNames.pr_with_global_universes i
```
is.

Reply:
what difference?

Reply:
sorry updated.